src/components/FormImage/FormImage.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+    <label
+        class="form-image"
+        :for="id"
+        :style="style"
+        @drop="onDrop"
+        @dragenter="cancelDrag"
+        @dragover="cancelDrag"
```

Comment:
```suggestion
        @dragenter.prevent
        @dragover.prevent
```

Is this not working?

Reply:
Nice one. Yes it works.